City-of-Helsinki/drupal-helfi-kymp
# UHF-2478: genesys chat cookie settings for eu_cookie_compliance

Adds Genesys chat cookie settings to EU Cookie Compliance.

How to test:

- set up KYMP instance
- checkout this branch: `git co UHF-2478_genesys_kymp_changes`
- import config: `make drush-cim`
- go to EU Cookie Compliance settings (`/admin/config/system/eu-cookie-compliance/settings`) and make sure that `essential:_genesys.widgets.*` is in the "Allowed cookies" list
- go to Cookie categories (`admin/config/system/eu-cookie-compliance/categories`) and make sure that there is a line for `_genesys.widgets.*` in the "Essential cookies" category
  - also see the language versions (fi and sv) and see that the lines are there

Also, see  the Platform Config PR that introduces these config changes: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/226
